whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/LirMntByAttributeCountValidator.java

```diff
@@ -37,7 +36,7 @@ public LirMntByAttributeCountValidator(final Maintainers maintainers) {
     @Override
     public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
         final RpslObject originalObject = update.getReferenceObject();
-        if (!LIR.equals(originalObject.getValueForAttribute(AttributeType.ORG_TYPE))) {
+        if (! OrgType.LIR.getName().equals(originalObject.getValueForAttribute(AttributeType.ORG_TYPE))) {
```

Comment:
LIR == OrgType.getFor(originalObject.getValueForAttribute(AttributeType.ORG_TYPE)) is a bit cleaner..?

Reply:
Agreed, refactored in all validators to be consistent.